### PR TITLE
Implement enviroment setup for serverside with a Get Route setup for …

### PR DESCRIPTION
**Pull request by jx114**
Implement env set up; package.json with the required dev dependencies, server connection to MongoDB connection.
Create seeding file: seeder.js and run with script, "npm run db:setup".
Implement get route for /api/galleries

### CODE REVIEW

File: database/seeder.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const faker = require('faker');
+const { Gallery } = require('./index.js');
+
+
+const randomImageUrl = 'https://loremflickr.com/1080/720/real_estate';
```
Comment (brandon-win) [inline database/seeder.js:4]: const variable that generates random image every time it is clicked

File: database/seeder.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    title: faker.lorem.sentence(),
+    reviews: Math.floor(Math.random() * (50 - 10) + 10),
+    rating: Math.random() * (5.0 - 4.0) + 4,
+    isSuperhost: faker.random.boolean(),
+    location: {
+      city: faker.address.city(),
+      state: faker.address.state(),
+      country: faker.address.country(),
+    },
+    gallery: galleries,
```
Comment (brandon-win) [inline database/seeder.js]: This currently takes in one galleries array, and is inputting the same array into all 100 data sets. Perhaps, create a method that generates a random amount of pictures in your gallery array (putting the gallery maker in a function)

**Review by brandon-win — CHANGES_REQUESTED** [review]
Please look at review of database/seeder.js

File: database/seeder.js (hunk 2 above)
Comment (jx114) [reply database/seeder.js]: Will do!

**Comment by jx114** [comment]
Making changes!

**Review by brandon-win — APPROVED** [review]

**Pull request merged**